test/runtests.jl

```diff
@@ -216,7 +223,11 @@ for useio in (false, true)
                 @test x == case.x
             end
             @test code == case.code
-            @test tlen == case.tlen
+            if Parsers.invalidquotedfield(code) || Parsers.quoted(code)
+                @test tlen == length(str)
```

Comment:
I _think_ this is correct 😅 

we can't harcode `case.tlen` in these cases now that we might be adding more characters to the string (e.g. when replace `{` => `{{` as the quote chars)